js/index.js

```diff
@@ -38,5 +38,64 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
+//header
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+let header = document.querySelectorAll('header nav a')
+header[0].setAttribute('class', 'nav-item-1')
+header[0].textContent = 'Services'
+header[1].setAttribute('class', 'nav-item-2')
+header[1].textContent = 'Product'
+header[2].setAttribute('class', 'nav-item-3')
+header[2].textContent = 'Vision'
+header[3].setAttribute('class', 'nav-item-4')
+header[3].textContent = 'Features'
+header[4].setAttribute('class', 'nav-item-5')
+header[4].textContent = 'About'
+header[5].setAttribute('class', 'nav-item-6')
```

Comment:
Good usage of `setAttribute`, but you could make it dynamic by using a `forEach` and a template literal.